src/main/java/org/fungover/storm/server/Server.java

```diff
@@ -40,10 +42,13 @@ public void acceptConnections(ServerSocket serverSocket) {
     }
 
     public static void main(String[] args) {
+        ConfigurationManager.getInstance().readConfigurationFile("src/main/resources/config.json");
+        Configuration conf = ConfigurationManager.getInstance().getCurrentConfiguration();
+
         LOGGER.info("Starting server...");
-        Map<String, String> env = System.getenv();
```

Comment:
Should the program select the env variable first then if it can't run it use the config file instead? or is it just inappropriate to have the config file there and I should just remove it from the method?

Reply:
> Should the program select the env variable first then if it can't run it use the config file instead? or is it just inappropriate to have the config file there and I should just remove it from the method?

Exactely as you said there. First try to see if env exists if not use config and last default